class.jetpack.php

```diff
@@ -1607,7 +1607,7 @@ public static function load_modules() {
 				continue;
 			}
 
-			if ( ! @include( Jetpack::get_module_path( $module ) ) ) {
+			if ( ! include_once( Jetpack::get_module_path( $module ) ) ) {
```

Comment:
Does `include_once()` return true or false if the file was already included somewhere else?